doc/world_format.txt

```diff
@@ -145,6 +145,16 @@ For load_mod_<mod>, the possible values are:
     * Other locations and absolute paths are not supported
     * Note that `moddir` is the directory name, not the mod name specified in mod.conf.
 
+PostgreSQL backend specific settings:
+  pgsqlXXXX_connection = host=127.0.0.1 port=5432 user=mt_user password=mt_password dbname=minetestXXXX
+  ^ XXXX stands for the used backend type, i.e. "", "player", "readonly" or "auth"
+
+Redis backend specific settings:
+  redis_address = 127.0.0.1  - Redis server address
```

Comment:
This indentation seems odd to me (2 spaces?), MD uses 4 spaces for code.

Reply:
Consistency with the documentation above